sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/parquet/ParquetFilters.scala

```diff
@@ -617,6 +623,10 @@ class ParquetFilters(
     nameToParquetField.contains(name) && valueCanMakeFilterOn(name, value)
   }
 
+  private def canMakeFilterOnPart(filter: sources.Filter): Boolean = {
```

Comment:
Seems to me this method name is a bit misleading. We are not really making a Parquet Filter for this partition filter and push it down to Parquet. It might be better to call it something like `canDynamicallyPushDownDataFilter`.